src/rustdoc_gen.rs

```diff
@@ -441,13 +468,13 @@ fn bytes2str(b: &[u8]) -> &std::ffi::OsStr {
     std::ffi::OsStr::new(str::from_utf8(b).unwrap())
 }
 
-pub(crate) struct RegistryBaseline {
+pub(crate) struct RustdocFromRegistry {
     target_root: PathBuf,
     version: Option<semver::Version>,
     index: crates_index::Index,
 }
```

Comment:
Is there a way to add a `derive Debug` to this struct when `index` doesn't implement this trait? Like, implementing the `Debug` for all variables besides `index`?

Reply:
Yes, you just need to write a manual `impl Debug for RustdocFromRegistry`.

Reply:
Done in 4113d4f07681e03a09c98c8edd9e65b0d52c98d8.